lib/auth/cognito.ts

```diff
@@ -107,14 +97,28 @@ export const initiateSignIn = async ({
       e instanceof CognitoIdentityProviderServiceException &&
       e.name === "NotAuthorizedException" &&
       e.message === "Password attempts exceeded"
-    )
+    ) {
+      const prismaUser = await prisma.user
+        .findUnique({
+          where: {
+            email: username,
+          },
+          select: {
+            id: true,
+          },
+        })
+        .catch((e) => prismaErrors(e, null));
+
       logEvent(
-        prismaUser?.accounts[0].id ?? "unknown",
-        { type: "User", id: prismaUser?.accounts[0].id ?? "unknown" },
+        prismaUser?.id ?? "unknown",
+        { type: "User", id: prismaUser?.id ?? "unknown" },
         "UserTooManyFailedAttempts",
         `Password attempts exceeded for ${username}`
       );
 
+      logMessage.warn("Cognito Lockout: Password attempts exceeded");
```

Comment:
Alarm already exist in CloudWatch but cannot work because of missing `Cognito Lockout: Password attempts exceeded` log.